translations/german.json

```
@@ -124,21 +124,21 @@
 		"The announcement was ended by ${user.name}.": "Die Ank&uuml;ndigung wurde von ${user.name} beendet.",
 		"Accepts the following commands:": "Folgende Befehle werden akzeptiert.",
 
-		"That option is not selected.": "",
-		"You have already voted for this poll.": "",
-		"No options selected.": "",
-		"you will not be able to vote after viewing results": "",
-		"View results": "",
-		"You can't vote after viewing results": "",
-		"The poll has ended &ndash; scroll down to see the results": "",
-		"Vote for ${num}": "",
-		"Submit your vote": "",
-		"Quiz": "",
-		"Poll": "",
-		"Submit": "",
-		"ended": "",
-		"votes": "",
-		"delete": "",
+		"That option is not selected.": "Diese Wahloption ist nicht ausgew&auml;hlt.",
+		"You have already voted for this poll.": "Du hast bereits f&uuml;r diese Umfrage abgestimmt.",
+	  "No options selected.": "Keine Wahloption wurde ausgew&auml;hlt.",
```

```suggestion
		"No options selected.": "Keine Wahloption wurde ausgew&auml;hlt.",
```

I believe you messed up the indentation for this one.